src/Http/Http/src/StreamPipeReaderOptions.cs

```diff
@@ -8,17 +8,17 @@
 
 namespace System.IO.Pipelines
 {
-    public class StreamPipeReaderOptions
+    public class StreamPipeReaderAdapterOptions
```

Comment:
Rename the file too

Reply:
Not going to bother. It's going to be deleted within a week (these are temporary changes) and I want to merge it before preview4 branching.